source/plugins/music/index.mjs

```diff
@@ -224,6 +247,70 @@ export default async function({login, imports, data, q, account}, {enabled = fal
             }
             break
           }
+          case "youtube": {
+            //Prepare credentials
+            let date = new Date().getTime()
+            let [, cookie] = token.split("; ").find(part => part.startsWith("SAPISID=")).split("=")
+            let sha1 = str => crypto.createHash("sha1").update(str).digest("hex")
+            let SAPISIDHASH = `SAPISIDHASH ${date}_${sha1(`${date} ${cookie} https://music.youtube.com`)}`
+            //API call and parse tracklist
+            try {
+              //Request access token
+              console.debug(`metrics/compute/${login}/plugins > music > requesting access token with youtube refresh token`)
+              const res = await imports.axios.post("https://music.youtube.com/youtubei/v1/browse?alt=json&key=AIzaSyC9XL3ZjWddXya6X74dJoCTL-WEYFDNX30",
```

Comment:
@rhld16 Sorry for noticing it only now, but shouldn't this key be a secret 😅 ?

Reply:
No, it's fine. This is the API key used by the YouTube Music browser for all users. You can easily find this key yourself by looking at any `https://music.youtube.com/youtubei` request even when logged out. The only sensitive info is the SAPIDHASH which is contained within the Cookie secret.

Reply:
Oki thanks a lot for replying and clarifying !
It's just it popped in when I enabled the secret scanning on this repo so I wanted to be sure you didn't leak your own key